ga4gh/exceptions.py

```diff
@@ -72,6 +71,21 @@ def getErrorCode(cls):
         code = zlib.crc32(cls.__name__) & 0xffffffff
         return code
 
+
+#####################################################################
+#
+# Exceptions that occur in the normal operation of the server
+#
+#####################################################################
+
+
+class RuntimeException(BaseServerException):
+    """
+    exceptions that can occur during the processing of a client request
+    """
+    httpStatus = -1
+    message = "Bad request"
```

Comment:
change this message to be something like BaseServerException's, since we should never be throwing it